src/qt/optionsdialog.cpp

```diff
@@ -293,12 +293,21 @@ void OptionsDialog::on_resetButton_clicked()
     }
 }
 
+
```

Comment:
Why this extra line?

Reply:
> Why this extra line?

Sorry, that was a blunder on my part. Fixed in 25bee9251374fa78d19077f2fb5fa0bfc5d651a2